front/src/WebRtc/MediaManager.ts

```diff
@@ -18,6 +18,12 @@ let videoConstraint: boolean|MediaTrackConstraints = {
     resizeMode: 'crop-and-scale',
     aspectRatio: 1.777777778
 };
+const audioConstraint: boolean|MediaTrackConstraints = {
```

Comment:
Can't you just type it as MediaTrackConstraints?bool should not be necessary anymore

Reply:
I don't know TypeScript well enough